successfactors-connector-demo/config/variables.yaml

```diff
@@ -0,0 +1,18 @@
+# == Variables ==
+# 
+# You can define here your project Variables.
+# If you want to define/override a Variable for a specific Environment, 
+# add an additional ‘variables.yaml’ file in a subdirectory in the ‘Config’ folder: 
+# '<project>/Config/_<environment>/variables.yaml
+#
+Variables:
+#  myVariable: value
+
+  successfactors-connector:
+  
+    host: '<myHost>'
+    
+    username: '<myUser>'
+    
+    # [password]
+    password: '<myPassword>'
```

Comment:
do not fill invalid values here: just keep em blank `host: ''` ... otherwise one must unpack this demo-project into the workspace, just to get rid of these bound and invalid values.